setup.py

```diff
@@ -1,11 +1,22 @@
-"""CodeJail: manages execution of untrusted code in secure sandboxes."""
+"""
+CodeJail: manages execution of untrusted code in secure sandboxes.
+"""
 
 from setuptools import setup
 
 setup(
     name="codejail",
-    version="0.1",
+    version="1.0",
     packages=['codejail'],
+    description="Manages execution of untrusted code in secure sandboxes.",
+    install_requires=[
+        'pylint==1.5.6'
```

Comment:
I think pylint (and nose and mock below) could be in a dev-requirements.txt or something like that.  The main point here is that users of CodeJail should be able to install it without automatically pulling in pylint.  I don't know what uses `tests_require`, I don't use it in my own projects.